Sewer56.Patcher.Regravitified.Te/Utility/Xor.cs

```diff
@@ -0,0 +1,162 @@
+﻿#if SRDXSelfContained
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics.X86;
+using System.Text;
+
+namespace Sewer56.Patcher.Riders.Dx.Utility;
+
+/// <summary>
+/// Implementation of XOR over a 32-byte array.
+/// </summary>
+public static class Xor
```

Comment:
The SIMD'ed XOR and Compression routines were used originally by the self contained (a.k.a. fake) patcher. Pretty sure this would be dead code right now.

Reply:
Was a copy-over from the folder, can remove easily. Do you suggest removing the Utility folder as a whole as well?